README.md

```diff
@@ -20,7 +20,45 @@ Make sure you are not introducing html errors or bad links:
 ```bash
 docker run --rm --label=jekyll --volume=$(pwd):/srv/jekyll  -it  jekyll/jekyll sh -c "bundle install && rake test"
 ```
+Should output:
 ```
 HTML-Proofer finished successfully.
 ```
+#### Side note for those on non-linux machines
+ 
+If you're developing locally but not on a Linux machine, you have a couple options. 
+You can opt to use [Docker for Mac](https://docs.docker.com/docker-for-mac/) / [Docker for Windows](https://docs.docker.com/docker-for-windows/). This will give you a docker environment from which to run the above docker container (which has all the of the correct Jekyll dependencies and Ruby versions installed). 
 
+
+If doing Istio development on Kubernetes and doing so locally with  [minikube](https://kubernetes.io/docs/getting-started-guides/minikube/)we highly recommend you use your OS's native virtualization. For example, to start minikube with native virtualization for Mac OSX try:
```

Comment:
This sounds like general advice that should be on the docs or in istio but maybe not in this file which is more about how to update/run the website?